### PR TITLE
Fix bootswatch theme name typo morth -> morph

### PR DESCRIPTION
**Pull request by r1gm**

### CODE REVIEW

**Comment by solomax** [comment]
Hello @r1gm 
These themes are being updated via script
Would this script work after your changes? :)

**Comment by r1gm** [comment]
i download the morph one from the script(i run the script in Windows WSL Ubuntu, i don't have real Linux)

**Comment by solomax** [comment]
Great, thanks for checking :)

File: bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss [hunk 1]
```diff
@@ -1,4 +0,0 @@
-@import "../../bootstrap/css/bootstrap.scss";
-@import "bootstrap.morth.variables.scss";
```
Comment (martin-g) [inline bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss:2]: I wonder how it worked until now ?!
The old / wrongly named file has some contents ...

It has been added at 2021 with https://github.com/martin-g/wicket-bootstrap/commit/bb7068ddef894123dfe511f6fbef57be9a9cbd5a#diff-683cc9e10b8f8a68e6ca651d4a8471ca5f992de071eccea96a257ba40a9cf206
and I see no related issue/PR for renaming from `morth` to `morph` at https://github.com/thomaspark/bootswatch/issues?q=is%3Aissue+morph+is%3Aclosed

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @r1gm !

File: bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss (hunk 1 above)
Comment (solomax) [reply bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss:2]: Maybe I did some sort of manual steps with typo ...
I can't remember now :(

Comment (martin-g) [reply bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss:2]: The contents is actually added by gettheme.sh itself ...

Comment (martin-g) [reply bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/bootswatch/css/bootstrap.morth.scss:2]: I've improved the Shell script to fail early with 6c54b09b2328727ccb06ab0aa05d04df1b2ff683

**Pull request merged**